packages/app/app/actions/importfavs.js

```diff
@@ -0,0 +1,20 @@
+import { LastFmApi } from '@nuclear/core';
+import { store } from '../persistence/store';
+import globals from '../globals';
+const lastfm = new LastFmApi(globals.lastfmApiKey, globals.lastfmApiSecret);
+// first function is used to extract the total number of loved tracks by sending a request to the
+// last.fm api for 1 song. (total number of loved tracks is included in  the response fields)
+// that number is then used to fetch all the songs in the second function which also adds them to the favorites
+export function ImportFmFavorites() {
```

Comment:
This function isn't needed, since you can use fetchAllFmFavorites with totalTracks=1